biotech-agentic-analyst/utils.py

```
@@ -0,0 +1,18 @@
+
+from PIL import Image
+
+
+def decode_thumbnail(b64_str: Optional[str]) -> Optional[Image.Image]:
+    """Decode a base64 string to an image."""
+    if not b64_str:
+        return None
+    try:
+        data = base64.b64decode(b64_str)
+        return Image.open(io.BytesIO(data))
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)